pages/dashboard/index.tsx

```diff
@@ -1,9 +1,13 @@
-import { NextPageWithLayout } from "../../util/page";
+import { NextPageWithLayout } from "@/util/page";
+import CustomRedirectButton from "@/components/util/CustomRedirectButton";
 import { PrimaryLayoutWithSidebar } from "@/components/layouts/exporter";
+
 const Dashboard: NextPageWithLayout = () => {
   return (
-    <section className="w-full h-screen bg-white">
+    <section className="w-full h-screen bg-white grid">
       This is the home page ui design for the dashboard
```

Comment:
replace the text with the given text in issue also place the text in center above the button

Reply:
Ohh, I forgot that.
I have changed the text and also made the content center.
Please review it now.